### PR TITLE
Updated for product names and versions

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://jira.coreos.com/browse/OSDOCS-697

cc @vikram-redhat

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@chrisnegus @kalexand-rh Do you mind eyeballing these updates to help me verify some of these version number bumps?

**Comment by vikram-redhat** [comment]
@openshift/team-documentation PTAL. The whole team should eyeball this and suggest changes for their content that might need updating with version numbers for 4.2 release.

File: modules/nw-install-config-parameters.adoc [hunk 1]
```diff
@@ -23,7 +23,7 @@ You cannot modify these parameters after installation.
 |`networking.networkType`
 |The network plug-in to deploy. The `OpenShiftSDN` plug-in is the only plug-in
 supported in {product-title} {product-version}. The `OVNKubernetes` plug-in is
-available as a Tech Preview in {product-title} 4.2.
+available as a Tech Preview in {product-title} {product-version}.
```
Comment (jboxman) [inline modules/nw-install-config-parameters.adoc]: But it will be GA in 4.3; Does it make sense to use the variable instead here?

File: modules/update-upgrading-web.adoc [hunk 2]
```diff
@@ -19,11 +19,11 @@ of the Customer Portal.
 
 . From the web console, click *Administration* > *Cluster Settings* and review
 the contents of the *Overview* tab.
-.. For production clusters, ensure that the *CHANNEL* is set to `stable-4.1`.
+.. For production clusters, ensure that the *CHANNEL* is set to `stable-4.2`.
```
Comment (kalexand-rh) [inline modules/update-upgrading-web.adoc]: Please revert the changes in this file - I have a change in flight to address this along with some other update issues.

File: modules/update-upgrading-cli.adoc [hunk 3]
```diff
@@ -84,7 +84,7 @@ previous command.
 $ oc get clusterversion -o json|jq ".items[0].spec"
 
 {
-  "channel": "stable-4.1",
+  "channel": "stable-4.2",
   "clusterID": "990f7ab8-109b-4c95-8480-2bd1deec55ff",
   "desiredUpdate": {
     "force": false,
```
Comment (kalexand-rh) [inline modules/update-upgrading-cli.adoc]: Please change line 92 to include "4.2.1"

File: modules/nw-install-config-parameters.adoc (hunk 1 above)
Comment (ahardin-rh) [reply modules/nw-install-config-parameters.adoc]: Just in case, I switched it back. Thanks!

File: modules/images-other-jenkins-agent-images.adoc [hunk 4]
```diff
@@ -11,10 +11,10 @@ The {product-title} Jenkins agent images are available on `quay.io` or
 Jenkins images are available through the Red Hat Registry:
 
 ----
-$ docker pull registry.redhat.io/openshift4/ose-jenkins:<v4.1.4>
-$ docker pull registry.redhat.io/openshift4/ose-jenkins-agent-nodejs:<v4.1.4>
-$ docker pull registry.redhat.io/openshift4/ose-jenkins-agent-maven:<v4.1.4>
-$ docker pull registry.redhat.io/openshift4/ose-jenkins-agent-base:<v4.1.4>
+$ docker pull regsitry.redhat.io/openshift4/ose-jenkins:<v4.2.0>
```
Comment (bergerhoffer) [inline modules/images-other-jenkins-agent-images.adoc]: Is there a reason that registry was changed to a misspelling of regsitry here? (And another instance in a file below)

Comment (ahardin-rh) [reply modules/images-other-jenkins-agent-images.adoc]: @bergerhoffer Good catch! Not sure what happened there, but all fixed.

**Comment by vikram-redhat** [comment]
@ahardin-rh I am going ahead and merging this. If there are any more changes required, can you create another PR?
